websites/S/Sezonluk Dizi/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
```

Comment:
please use latest schema (v1.3)

Reply:
This is an old commit I guess. It's already set to 1.3